google_cloud_vision_msgs/BoundingPoly bounding_poly
google_cloud_vision_msgs/BoundingPoly fd_bounding_poly
google_cloud_vision_msgs/Landmark[] landmarks
float32 roll_angle
float32 pan_angle
float32 tilt_angle
float32 detection_confidence
float32 landmarking_confidence
float32 joy_likelihood
float32 sorrow_likelihood
float32 anger_likelihood
float32 surprise_likelihood
float32 under_exposed_likelihood
float32 blurred_likelihood
float32 headwear_likelihood